src/Arc/Clay/App.hs

```diff
@@ -15,28 +16,17 @@ import Clay.Stylesheet (key)
 appStyle :: Css
 appStyle = do
     normalise
-    --star ? globalStyle
     body ? bodyStyle
     fonts
     fieldset_
     forms
     icons
+    messages
     layouts
     iconWithTexts
     inputs
     buttons
 
-globalStyle :: Css
-globalStyle =
-    star ? do
-        key "appearance" $ Value "none"
-        key "font" $ Value "inherit"
-        key "color" $ Value "inherit"
-        key "cursor" $ Value "inherit"
-        key "highlight" $ Value "none"
-        key "outline" $ Value "none"
-        boxSizing borderBox
-
 bodyStyle :: Css
```

Comment:
Now with colour tokens, we should apply foreground and background colours to `body` as well. This will make it easier to implement prefers-color-scheme in the future.